dataset/test/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ add_executable(
   dataset_test.cpp
   dataset_view_test.cpp
   data_view_test.cpp
-  event_data_operations_consistency_test.cpp
+  # event_data_operations_consistency_test.cpp
```

Comment:
Commented code?

Reply:
I left it as a reminder that those files have not been removed yet.